### PR TITLE
richdocuments: add host_session_id parameter

### PR DESCRIPTION
**Pull request by hcvcastro**
* Resolves: # 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by juliushaertl** [comment]
Can you provide some context on what this is used?

In addition you can already get the richdocuments version in the frontend code through the capabilities so no backend changes should be required and please also add it to https://github.com/nextcloud/richdocuments/blob/e9ec3189c92c229475f6430b47e02d6463699c65/src/view/Office.vue#L51
```
import { getCapabilities } from '../services/capabilities.ts'

getCapabilities().version
```

**Review by juliushaertl — CHANGES_REQUESTED** [review]
Needs mentioned adjustments

**Comment by hcvcastro** [comment]
Thanks, it is mostly to implement Wopi HOST_SESSION_ID, to debugging log purposes and
to track wopi host  logs for troubleshooting purposes based on a host-specific session identifier

File: src/document.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ import '../css/document.scss'
 import axios from '@nextcloud/axios'
 import { spawnDialog } from '@nextcloud/dialogs'
 import SaveAs from './components/Modal/SaveAs.vue'
+import { getCapabilities } from '../services/capabilities.ts'
```
Comment (juliushaertl) [inline src/document.js:24]:
```suggestion
import { getCapabilities } from './services/capabilities.ts'
```

**Review by juliushaertl — CHANGES_REQUESTED** [review]
Still needs to also be added in 
https://github.com/nextcloud/richdocuments/blob/e9ec3189c92c229475f6430b47e02d6463699c65/src/view/Office.vue#L51

**Pull request merged**